packages/mui-codemod/src/v5.0.0/date-pickers-moved-to-x.js

```diff
@@ -0,0 +1,236 @@
+const exports = {
```

Comment:
Is it intentional that you are not listing the `Adapter*` imports? Are we not going to export them from the x repository? If yes, we should list this as a breaking change, if not we should include them.

Reply:
No, I only listed the elements exported from the root and forgot the other
But I did migrate them.

Reply:
Ok, let's add them in that case. Maybe we should also throw a warning if someone accidently imported something from `@mui/lab/internal`, but up to you, it's anyway not allowed.

Reply:
Done :+1: